metadata-ingestion/src/datahub/integrations/great_expectations/action.py

```diff
@@ -680,7 +688,12 @@ def make_dataset_urn_from_sqlalchemy_uri(
                     for {data_platform}."
             )
             return None
-        schema_name = "{}.{}".format(url_instance.database, schema_name)
+        schema_name = "{}.{}".format(
+            url_instance.database.lower()
+            if data_platform == "snowflake"
```

Comment:
Thanks. We might want to drop a pointer to the code link in a comment here. Will append this into the PR